src/cache_cf.cc

```diff
@@ -4495,6 +4489,10 @@ static void parse_sslproxy_cert_adapt(sslproxy_cert_adapt **cert_adapt)
             }
             ca->param = xstrdup(param);
         }
+
+    } else if (ca->alg != Security::algSetEnd) {
```

Comment:
FYI: This condition will go away once certAdaptAlgorithmId() stops returning an invalid algorithm value.

Reply:
done